src/System.Text.RegularExpressions/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Text.RegularExpressions:
+MembersMustExist : Member 'System.Text.RegularExpressions.RegexOptions System.Text.RegularExpressions.RegexOptions.Debug' does not exist in the implementation but it does exist in the contract.
```

Comment:
This is debug only.

Reply:
Should the tool run against release builds instead?

Reply:
It will run on both it is OK for this to be in the baseline for both it won't hurt anything.